### DISCUSSION TITLE
Firebase analytics should report screen_resolution internally.

### DISCUSSION THREAD

**Discussion by BoKKeR** [invertase/react-native-firebase#5638] [Ideas]
I have seen this mentioned on the old archived repo, but I cant find any mention of it on this repo or in the code. https://github.com/rnc-archive/react-native-google-analytics/issues/39

The screen_resolution is never set when using RNFirebase.analytics, is it a forgotten feature or is it something that is blocking this? 

I didnt see any code in the repo mentioning resolution.

```
    "@react-native-firebase/analytics": "10.8.1",
    "@react-native-firebase/app": "10.8.1",
    "@react-native-firebase/database": "10.8.1",
    "@react-native-firebase/dynamic-links": "10.8.1",
    "@react-native-firebase/messaging": "10.8.1",
```

**Comment by mikehardy** [comment]
Hi there!
Those versions are pretty old, you should update

I can't reproduce though, I have screen resolution in my analytics for one of my work projects

![Screenshot from 2021-08-23 12-28-47](https://user-images.githubusercontent.com/782704/130491039-550bdf95-b12c-4e94-8792-28777887a31a.png)

**Comment by BoKKeR** [comment]
I updated to the 12.7.2. Did some events but the resolution didnt show up after 24h.
I also tried the firebase debug mode, where the property does not show up. Should it show up there?

**Reply by mikehardy** [reply]
I don't know, all I can say is I can't reproduce. I know "it works for me" is frustrating to hear, but if I can't reproduce, I can't see a problem, and without seeing a problem I can't fix it. Sorry

**Reply by BoKKeR** [reply]
Can you confirm if the property shows up in firebase debug mode for you?

**Reply by mikehardy** [reply]
I cannot confirm this - I was checking other analytics items and examined this item. In DebugView there is nothing related to demographics or technology (device type etc), it's simply not there. However those things are being recorded and you should be able to see them if you link firebase analytics with google analytics and go over there. I do see on Firebase Dashboard that the "technology" tab of the default "what is your audience like?" is populated with device information. I don't think there is an issue here
